health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/pages/employee/index.js

```
@@ -0,0 +1,252 @@
+    url: `/boundary-service/boundary-hierarchy-definition/_search`,
+    changeQueryName: `${BOUNDARY_HIERARCHY_TYPE}`,
+    body: {
+      BoundaryTypeHierarchySearchCriteria: {
+        tenantId: tenantId,
+        limit: 1,
+        offset: 0,
+        hierarchyType: BOUNDARY_HIERARCHY_TYPE,
+      },
+    },
+    config: {
+      cacheTime: Infinity,
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)